### ISSUE TITLE
Speech to text workflow - Open kiosk client throws error

### ISSUE THREAD

**Issue by Rocketman17** [GenAI-Nerds/Gapi#9]
Open kiosk client throws error if we say something - Can't find registered node in that workflow nodekey.

![image](https://github.com/user-attachments/assets/0200eb61-cb2e-44cd-9450-e5d608824083)

**Comment by GenAINerdKerry** [comment]
Yeah, good point, that thing won't work exactly when shipped. It was meant as a reference and should have docs on how to update the Nodes to make it work in "your" environment. Leaving open
